cms/plugin_rendering.py

```diff
@@ -472,6 +493,19 @@ def render_plugin(self, instance, context, placeholder=None, editable=False):
             placeholder_cache.setdefault('plugins', []).append(instance)
         return mark_safe(content)
 
+    def render_exception(self, action, instance, context, placeholder, editable):
+        if editable:
+            exc, value, traceback = context['exc_info']
+            return f'<div style="display: inline-block; border: none; background: rgba(0, 0, 0, 0.5); ' \
```

Comment:
Hi @fsbraun. Do you think it is a good idea to write it like this inside `"""`?

```python
>>> a = 1
>>> b = 2
>>> sentence = f"""
... <div>
... {a}: {b}
... </div>
... """
>>> sentence
'\n<div>\n1: 2\n</div>\n'>>> a = 1
>>> b = 2
>>> sentence = f"""
... <div>
... {a}: {b}
... </div>
... """
>>> sentence
'\n<div>\n1: 2\n</div>\n'
```

This should just work, except that you don't have to escape each line using `\`